test/runtests.py

```diff
@@ -64,6 +65,9 @@
     print('{} not found. Did you run ./build.sh already?'.format(binary))
     sys.exit(1)
 
+if len(sys.argv) > 1:
+    if sys.argv[1] in ['-?', '--help']:
+        print "ChakraCore *nix Test Script\n"
```

Comment:
Did you keep this intentionally, or was it merge error? This doesn't seem needed. The argparse module handles `-h, --help` automatically. It doesn't handle `-?` as I recall, but also prints help on that.